styles.css

```diff
@@ -1,11 +1,20 @@
 @import url('https://fonts.googleapis.com/css2?family=Alumni+Sans+Pinstripe&family=Poppins&display=swap');
 
+body {
+  background-color: white;
+}
+
 #toolbar {
   display: flex;
   position: fixed;
   height: 35px;
-  width: 920px;
+  width: 327px;
   padding: 10px;
+  background-color: white;
```

Comment:
- [ ] **[OPTIONAL]** Please, 🙏🏽 in `#toolbar` avoid using `position: fixed` it is for a better user experience.

![image](https://user-images.githubusercontent.com/58642949/185766961-2fe1a6bc-b548-4529-8021-14132282efa7.png)